packages/storage/src/resource_clients/request_queue.ts

```diff
@@ -230,6 +230,11 @@ export class RequestQueueClient {
         const orderNo = this._calculateOrderNo(request, forefront);
         const id = uniqueKeyToRequestId(request.uniqueKey);
         if (request.id && id !== request.id) throw new Error('Request ID does not match its uniqueKey.');
+        // @ts-ignore
+        request.userData = { ...request.userData, __crawlee: request.internalVariables };
+        // @ts-ignore
+        delete request.internalVariables;
```

Comment:
~let's use casting instead of ts-ignore~

actually, this package should not be touched at all, i guess this is just a residue from the original version, right?